tidb-cloud/backup-and-restore.md

```diff
@@ -14,25 +14,57 @@ This document describes how to back up and restore your TiDB cluster data on TiD
 
 ## Backup
 
-TiDB Cloud provides two types of data backup: automatic backup and manual backup.
+TiDB Cloud supports automatic backup and manual backup.
 
-Daily backups are automatically scheduled for your TiDB clusters in TiDB Cloud. You can pick a backup snapshot and restore it into a new TiDB cluster at any time. Automated backup can reduce your losses in extreme disaster situations.
+Automatic backups are scheduled for your TiDB clusters according to the backup setting, which can reduce your loss in extreme disaster situations. You can also pick a backup snapshot and restore it into a new TiDB cluster at any time.
 
 ### Automatic backup
 
 By the automatic backup, you can back up the cluster data every day at the backup time you have set. To set the backup time, perform the following steps:
 
 1. Navigate to the **Backup** tab of a cluster.
 
-2. Click **Auto Setting**. The setting window displays.
+2. Click **Backup Settings**. The setting window displays.
 
-3. In the setting window, select **Backup Time** and maximum backup files in **Limit (backup files)**.
+3. In the setting window, configure the automatic backup:
+
+    - (Optional) Check whether the PITR (**Point-in-time Recovery**) feature is on. 
```

Comment:
```suggestion
    - (Optional) Enable the PITR (**Point-in-time Recovery**) feature according to your needs. 
```

Reply:
No switch is implemented, only a status whether it is on or not.